src/views/List.jsx

```diff
@@ -12,35 +13,50 @@ export function List({ data }) {
 		item.name.toLowerCase().includes(searchItem.toLowerCase()),
 	);
 
+	const listName = listPath.slice(29);
+
 	return (
 		<>
-			<form onSubmit={(event) => event.preventDefault()}>
-				<label htmlFor="search-item">Search Item: </label>
-
-				<input
-					id="search-item"
-					type="search"
-					placeholder="Search Item..."
-					onChange={handleTextChange}
-					value={searchItem}
-				/>
-
-				{searchItem && (
-					<button
-						type="button"
-						onClick={() => {
-							setSearchItem('');
-						}}
-					>
-						X
-					</button>
-				)}
-			</form>
-			<ul>
-				{filteredItems.map((item) => {
-					return <ListItem key={item.id} name={item.name} />;
-				})}
-			</ul>
+			{!data.length ? (
+				<>
+					<p>Welcome to {listName}!</p>
+					<p>Ready to add your first item? Start adding below!</p>
+
+					<AddItems />
+				</>
+			) : (
+				<>
+					<p>{listName}</p>
+
+					<form onSubmit={(event) => event.preventDefault()}>
+						<label htmlFor="search-item">Search Item: </label>
+
+						<input
+							id="search-item"
+							type="search"
+							placeholder="Search Item..."
+							onChange={handleTextChange}
+							value={searchItem}
+						/>
+
+						{searchItem && (
+							<button
+								type="button"
+								onClick={() => {
+									setSearchItem('');
+								}}
+							>
+								X
+							</button>
+						)}
```

Comment:
We should consider removing the rendering of this extra button since the input has the functionality to allow the user to clear the output!